### PR TITLE
Fixed auto-evo prediction panel overlapping the organism statistics panel when overflowed

### PR DESCRIPTION
**Pull request by Kasterisk**
**Brief Description of What This PR Does**

**Related Issues**

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [x] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author.

### CODE REVIEW

File: src/gui_common/thrive_theme.tres [hunk 1]
```diff
@@ -165,7 +165,7 @@ thickness = 5
 content_margin_left = 13.0
 content_margin_top = 2.0
 
-[sub_resource type="StyleBoxFlat" id=21]
+[sub_resource type="StyleBoxFlat" id=41]
```
Comment (hhyyrylainen) [inline src/gui_common/thrive_theme.tres:168]: These seem like incorrect ID changes. Are there any intended changes in this file?

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]

File: src/gui_common/thrive_theme.tres (hunk 1 above)
Comment (Kasterisk) [reply src/gui_common/thrive_theme.tres:168]: I've did git restore on this file but godot insisted on changing these IDs.

Comment (hhyyrylainen) [reply src/gui_common/thrive_theme.tres:168]: Doesn't do it for me. Testing on this branch, Godot automatically fixes the IDs for me:

Kuvakaappaus - 2021-11-21 20-23-35

Comment (hhyyrylainen) [reply src/gui_common/thrive_theme.tres:168]: Actually, I'll push that change to this branch, let me know if that causes problems.

**Review by hhyyrylainen — APPROVED** [review]
I don't see any more thrive theme changes in this PR. I think the microbe editor tscn changes are fine but there's way too many for me to check and think through what's been changed.

**Review by FuJa0815 — APPROVED** [review]

File: src/gui_common/thrive_theme.tres (hunk 1 above)
Comment (Kasterisk) [reply src/gui_common/thrive_theme.tres]: Looks good.

**Comment by Kasterisk** [comment]
This should also fix problem with the panels overlapping after changing the language to english from another language.

File: src/gui_common/thrive_theme.tres (hunk 1 above)
Comment (Kasterisk) [reply src/gui_common/thrive_theme.tres]: Scratch that, looks like this breaks the styling for species name's LineEdit. The thrive theme got automatically changed last time, so maybe my ID changes is the right one?

**Pull request merged**